### PR TITLE
Code Quality: Suppressed NullReferenceException caused by EnableRaisingEvents

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- https://files-org.sentry.io/issues/5497293589/events/97d0f7a7cda546d19e51e5ca2ebe4c3d/

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files ...
2. ...

### CODE REVIEW

**Comment by hishitetsu** [comment]
Is the crash report you mentioned correct? The link is to a FileNotFoundException.

**Comment by yaira2** [comment]
I'm not sure if it fully solves the issue, but some of the events in that report include mention of this issue, so this should definitely help.

File: src/Files.App/ViewModels/ShellViewModel.cs [hunk 1]
```diff
@@ -1956,7 +1960,7 @@ private void WatchForWin32FolderChanges(string? folderPath)
 				watcher.Deleted += DirectoryWatcher_Changed;
 				watcher.Renamed += DirectoryWatcher_Changed;
 				watcher.EnableRaisingEvents = true;
-			}
+			});
```
Comment (hishitetsu) [inline src/Files.App/ViewModels/ShellViewModel.cs:1963]:
```suggestion
			}, App.Logger);
```
Please set up the logger for debugging.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**